UI/Web/src/app/manga-reader/_components/infinite-scroller/infinite-scroller.component.ts

```diff
@@ -240,30 +240,37 @@ export class InfiniteScrollerComponent implements OnInit, OnChanges, OnDestroy,
       takeUntilDestroyed(this.destroyRef)
     );
 
+    // We need the injector as toSignal is only allowed in injection context
+    // https://angular.dev/guide/signals#injection-context
```

Comment:
Going forward, you don't need to include this as it's well known if you don't setup signals in the constructor then you need this.